scrambles/src/test/java/net/gnehzr/tnoodle/puzzle/SquareOnePuzzleTest.java

```diff
@@ -41,6 +40,6 @@ public void testSomething() throws InvalidMoveException {
         solution = state.solveIn(2);
         assertEquals(solution, "(-3,1)");
 
-        System.out.println(sq1.generateWcaScramble(Utils.getSeededRandom()));
+        System.out.println(sq1.generateScramble());
```

Comment:
Why the change here (and in 3x3 fewest moves test below)?

We went to a good bit of effort to make sure that tests were random, but reproducible so long as you use the same seed. It would be a shame to lose that.

Reply:
I was unaware of those efforts to be honest. We can revert this change at any given time if you feel it's necessary.